css/main.css

```diff
@@ -0,0 +1,964 @@
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+.icon-big {
+  font-size: 3em;
+  background-color: white;
+  color: black;
+  width: 100%;
+  border-radius: 50%;
+  padding: 1.25rem 1.25rem;;
+}
+
+.icon-cog:before {
+  content: "\e600";
+}
+
+.icon-lab:before {
+  content: "\e601";
+}
+
+.icon-brush:before {
+  content: "\e602";
+}
+
+.icon-pin:before {
+  content: "\e603";
+}
+
+.icon-mail:before {
+  content: "\e604";
+}
+
+.icon-mobile:before {
+  content: "\e605";
+}
+
+.s-icon-circle {
+  text-decoration: none;
+  color: #fd6632;
+  font-size: 3.25rem;
+  padding: 10%;
+  margin: auto;
+}
+
+.s-icon-circle:hover {
+  color: #0b9ce0;
+}
+
+.icon-circle-right {
+  order: 3;
+}
+
+.icon-circle-right:before {
+  content: "\e606";
+}
+
+.icon-circle-left {
+  order: 1;
+}
+
+.icon-circle-left:before {
+  content: "\e607";
+}
+
+.icon-social {
+  text-decoration: none;
+  font-size: 2rem;
+  padding: 1%;
+}
+
+.icon-google-plus2 {
+  color: #fabc00;
+}
+
+.icon-google-plus2:before {
+  content: "\e608";
+}
+
+.icon-facebook2 {
+  color: #3b5998;
+}
+
+.icon-facebook2:before {
+  content: "\e609";
+}
+
+.icon-twitter2 {
+  color: #00aced;
+}
+
+.icon-twitter2:before {
+  content: "\e60a";
+}
+
+.icon-dribbble2 {
+  color: #dd2667;
+}
+
+.icon-dribbble2:before {
+  content: "\e60b";
+}
+
+.icon-linkedin {
+  color: #007bb5;
+}
+
+.icon-linkedin:before {
+  content: "\e60c";
+}
+
+/*=== FONTS ===*/
+@font-face {
+  font-family: 'Helvetica';
+}
+
+@font-face {
+  font-family: 'open_sanssemibold';
+  src: url('opensans-semibold-webfont.eot');
+  src: url('opensans-semibold-webfont.eot?#iefix') format('embedded-opentype'),
+       url('opensans-semibold-webfont.woff2') format('woff2'),
+       url('opensans-semibold-webfont.woff') format('woff'),
+       url('opensans-semibold-webfont.ttf') format('truetype'),
+       url('opensans-semibold-webfont.svg#open_sanssemibold') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+
+/*=== LAYOUT ===*/
+body {
+  width: 100%;
+  margin: 0;
+  text-align: center;
+}
+
+.nav-toggle {
+  position: fixed;
+  left: 0;
+  width: 1em;
+  height: 1.25em;
+  font-size: 2em;
+  margin: .5em;
+  transition: left 0.3s ease;
+}
+
+.nav-toggle:before {
+  content: '☰';
+}
+
+.trigger {
+  position: fixed;
+  top: 0;
+  left: 15em;
+  width: 100%;
+  height: 100%;
+  pointer-events: none;
+  z-index: 1;
+}
+
+.l-nav {
+  position: fixed;
+  left: -15em;
+  width: 15em;
+  height: 100%;
+  background: #2e2e2e;
+  opacity: .95;
+  transition: left 0.3s ease;
+}
+
+.page-wrapper {
+  position: relative;
+  right: 0;
+  transition: right .3s ease;
+}
+
+.mobileNavToggle {
+  display: none;
+}
+
+.mobileNavToggle:checked ~ .l-nav {
+  left: 0;
+}
+
+.mobileNavToggle:checked ~ .icon-menu {
+  left: 15em;
+}
+
+.mobileNavToggle:checked ~ .trigger {
+  pointer-events: auto;
+}
+
+.mobileNavToggle:checked ~ .page-wrapper {
+  right: -15em;
+}
+
+/*=== DIVOTS ===*/
+.b-orange::before {
```

Comment:
I think I mentioned it before, but I want to make sure it's not missed - please try to avoid using presentation-specific names for classes. Inevitably this will change at some point and then be wrong.